core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -209,6 +219,13 @@ <h3>Advanced Features</h3>
               </button>
             </span>
             <span class="col-lg-8 col-md-8 col-sm-8 help-block" style="font-size: smaller;">
+              The feature is
```

Comment:
The --> This

Otherwise, PR looks good. I'll wait for you to get confirmation of the screenshots from Joe (as discussed before) before giving LGTM.

Reply:
Okay! I sent the screenshots to Joe, will inform you as soon as I get the feedback.

Reply:
Done!